meshtastic/interfaces/ble/management_runtime.py

```
@@ -0,0 +1,934 @@
+        """Return whether a candidate client appears currently connected."""
+        if client is None:
+            return False
+        for attr_name in ("isConnected", "is_connected", "_is_connected"):
+            is_connected = getattr(client, attr_name, None)
+            if callable(is_connected) and not _is_unconfigured_mock_callable(
+                is_connected
+            ):
+                try:
+                    return bool(is_connected())
+                except Exception:  # noqa: BLE001 - connectivity probe must remain best effort
+                    continue
```

Ruff (0.15.6)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)